### PR TITLE
Rebrandly #3755

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

File: components/rebrandly/rebrandly.app.mjs [hunk 1]
```diff
@@ -1,11 +1,101 @@
...
+          },
+        });
+
+        return {
+          options: domains.map((domain) => ({
+            label: domain.fullName,
+            value: domain.id,
+          })),
+          context: {
+            lastId: domains[domains.length].id,
```
Comment (andrewjschuang) [inline components/rebrandly/rebrandly.app.mjs:26]: Shouldn't this be `domains[domains.length - 1].id` ?
And if so, a check if `domains.length > 0` would be necessary to not make it `undefined`

File: components/rebrandly/rebrandly.app.mjs [hunk 2]
```diff
@@ -1,11 +1,101 @@
...
+          },
+        });
+
+        return {
+          options: links.map((link) => ({
+            label: link.title ?? link.destination,
+            value: link.id,
+          })),
+          context: {
+            lastId: links[0].id,
```
Comment (andrewjschuang) [inline components/rebrandly/rebrandly.app.mjs:48]: Shouldn't this be like `domains`?

File: components/rebrandly/rebrandly.app.mjs (hunk 1 above)
Comment (lcaresia) [reply components/rebrandly/rebrandly.app.mjs:26]: You are right, sorry!

File: components/rebrandly/rebrandly.app.mjs [hunk 3]
```diff
@@ -1,11 +1,105 @@
...
+
+        return {
+          options: domains.map((domain) => ({
+            label: domain.fullName,
+            value: domain.id,
+          })),
+          context: {
+            lastId: domains.length > 0
+              ? domains[domains.length - 1].id
+              : undefined,
```
Comment (andrewjschuang) [inline components/rebrandly/rebrandly.app.mjs:28]: If marking this as `undefined`, wouldn't it start loading again from the first page?

File: components/rebrandly/rebrandly.app.mjs [hunk 4]
```diff
@@ -1,11 +1,105 @@
...
+
+        return {
+          options: links.map((link) => ({
+            label: link.title ?? link.destination,
+            value: link.id,
+          })),
+          context: {
+            lastId: links.length > 0
+              ? links[links.length - 1].id
+              : undefined,
```
Comment (andrewjschuang) [inline components/rebrandly/rebrandly.app.mjs:52]: If marking this as `undefined`, wouldn't it start loading again from the first page?

**Review by andrewjschuang — COMMENTED** [review]
Thanks, LGTM!

**Comment by DilanAthukorala** [comment]
Verification completed, and ready for release.

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**